apps/prairielearn/src/tests/groupScoreAndSync.test.ts

```diff
@@ -25,7 +25,7 @@ const question = [{ qid: 'addNumbers', type: 'Freeform', maxPoints: 5 }];
 const questions = _.keyBy(question, 'qid');
 
 describe('assessment instance group synchronization test', function () {
-  this.timeout(10000);
+  this.timeout(20000);
```

Comment:
Not really sure why this specific PR impacts these times.

Reply:
This is a big red flag; this means that this (maybe, somehow) impacts Python execution times, which we really don't want to increase. This warrants further investigation.

Reply:
It looks like you're using uv to install Python itself now, and that seems to use this: https://github.com/astral-sh/python-build-standalone

Maybe this is less optimized than conda's build. You could use conda to install Python itself and then use uv to install additional libraries on top of that. I had experimented with that in the vscode workspace and it seemed to be working fine.

Reply:
https://github.com/PrairieLearn/PrairieLearn/pull/11039#issuecomment-2864314797